File: UVA/p12372.py
t = int(input())
for i in range(t):
    L, W, H = input().split()    
    if int(L)<=20 and int(W)<=20 and int(H)<=20:
        print('Case {}: good'.format(i+1))    
    else:
        print('Case {}: bad'.format(i+1))

File: UVA/p12372.java

import java.util.Scanner;

/**
 * AC, easy, ad hoc, 30 August 2018.
 * @author Faiz Ikhwan
 */
public class p12372 
{
    public static void main(String[] args)
    {
        Scanner input = new Scanner(System.in);
        int t = input.nextInt();
        
        for(int i=0; i<t; i++)
        {
            int L = input.nextInt();
            int W = input.nextInt();
            int H = input.nextInt();
            
            if(L<=20 && W<=20 && H<=20)
                System.out.println("Case "+(i+1)+": good");
            else
                System.out.println("Case "+(i+1)+": bad");
        }
    }
}
